.github/workflows/release.yml

```
@@ -77,13 +77,337 @@ jobs:
+
       - name: Create GitHub release
         env:
           GH_TOKEN: ${{ github.token }}
-        run: >
-          gh release create "${{ github.ref_name }}"
-          artifacts/WheelWizardWindows.exe
-          artifacts/WheelWizard_Linux
-          artifacts/WheelWizard_arm64_Linux
-          --title "${{ github.ref_name }}"
-          --generate-notes
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:6:49: Double quote to prevent globbing and word splitting

(shellcheck)